WorkerApi/tests/WorkerApiTests/UnitTest1.cs

```diff
@@ -1,82 +0,0 @@
-// This file is part of the ArmoniK project
```

Comment:
Wasn't this test used?

Reply:
It was not used in pipelines. I tried to run it but it did not work

Reply:
Wasn't used. And new unit test will come after test documentation